components/user-claim-retriever/src/main/java/com/wso2telco/claims/RemoteClaims.java

```diff
@@ -19,6 +19,6 @@
 
 public interface RemoteClaims {
 
```

Comment:
Add a comment explaining the usage of this interface.

Reply:
fixed